### ISSUE TITLE
Show Client Charge 

### ISSUE THREAD

**Issue by therajanmaurya** [openMF/self-service-app#61]
Self-Service is a end user application. So we have to show as much as data to client, So that client will be satisfied.

Currently UI is like 
![screenshot](https://cloud.githubusercontent.com/assets/5724482/20033167/e58cdc6e-a3c0-11e6-8908-53c7c7e602f1.jpg)

We want this UI
![group 6](https://cloud.githubusercontent.com/assets/5724482/20033169/ec180a90-a3c0-11e6-8151-87744260750b.png)

How to achieve this Issue : 

Change the ClientCharge Item UI

**Comment by sosnickm** [comment]
Rajan, could you tell me what are the id's for the values Due amount, Paid amount, waived amount and outstanding amount? (right now there's just one id called tv_amount)

**Comment by therajanmaurya** [comment]
You have to implement this UI from scratch.

**Comment by sosnickm** [comment]
<img width="403" alt="screen shot 2016-11-22 at 10 39 25 am" src="https://cloud.githubusercontent.com/assets/12140915/20529971/fb50d3d4-b09f-11e6-80e7-f652f719a021.png">

**Comment by sosnickm** [comment]
Rajan, this is what I have now, but I am unsure of two things. 
1 - Can you explain the green dot in the corner /  is it necessary?
2 - The 'Charge' class only has one number stored as amount. I don't see where I would get the information for "paid amount", "waived amount", or "outstanding amount". Where is that info stored?

**Comment by therajanmaurya** [comment]
_1 - Can you explain the green dot in the corner / is it necessary?_
Green dot showing the status of the charge, Is active or not. Yes, this is necessary.

_2 - The 'Charge' class only has one number stored as the amount. I don't see where I would get the information for "paid amount", "waived amount", or "outstanding amount". Where is that info stored?_
I have looked the source code, you have to change the ```Charge``` Pojo to complete this issue.
Here is the full POJO 
```json
{
      "id": 10,
      "clientId": 1,
      "chargeId": 13,
      "name": "dscs",
      "chargeTimeType": {
        "id": 2,
        "code": "chargeTimeType.specifiedDueDate",
        "value": "Specified due date"
      },
      "dueDate": [
        2017,
        12,
        30
      ],
      "chargeCalculationType": {
        "id": 1,
        "code": "chargeCalculationType.flat",
        "value": "Flat"
      },
      "currency": {
        "code": "USD",
        "name": "US Dollar",
        "decimalPlaces": 2,
        "displaySymbol": "$",
        "nameCode": "currency.USD",
        "displayLabel": "US Dollar ($)"
      },
      "amount": 200,
      "amountPaid": 0,
      "amountWaived": 0,
      "amountWrittenOff": 0,
      "amountOutstanding": 200,
      "penalty": false,
      "isActive": true,
      "isPaid": false,
      "isWaived": false
    }
```

**Comment by sosnickm** [comment]
We should close this issue as the PR was accepted

**Comment by therajanmaurya** [comment]
fix in #74
